src/options/set.rs

```diff
@@ -515,18 +516,70 @@ fn parse_paging_mode(paging_mode_string: &str) -> PagingMode {
     }
 }
 
+fn parse_width_plusminus(width_arg: &str, terminal_width: usize) -> Result<usize, String> {
```

Comment:
The name "plusminus" is similar to the name "minusplus" that you're introducing in the main PR. Shall we use something different here, such as `parse_width_specifier()`?